overlord/ifacestate/helpers_test.go

```diff
@@ -129,6 +132,7 @@ func (s *helpersSuite) TestGetConns(c *C) {
 		c.Assert(id, Equals, "APP:network CORE:network")
 		c.Assert(connState.Auto, Equals, true)
 		c.Assert(connState.Interface, Equals, "network")
+		c.Assert(connState.StaticSlotAttrs["number"], Equals, int(78))
```

Comment:
This failed on unit tests, the failure seems to suggest that we need to do some more mapping from `json.Number` to what the interface code understands.

```
FAIL: helpers_test.go:113: helpersSuite.TestGetConns

helpers_test.go:135:
    c.Assert(connState.StaticSlotAttrs["number"], Equals, int(78))
... obtained json.Number = "78"
... expected int = 78
```